test/run-mocha.js

```diff
@@ -1,10 +0,0 @@
-/**
```

Comment:
The purpose of this file was to enable Mocha integration with IDEs. We were able to run Mocha tests directly inside WebStorm with it.

Later I noticed that some of the logic inside `load-suite` has changed and broke this. I'd love see this working again but for the time being I think it's safe to remove this file since it's not working.